grouped-symbol-column/index.html

```diff
@@ -0,0 +1,38 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Grouped symbol chart - Visual Vocabulary</title>
+    <script src="//unpkg.com/@financial-times/d3-bootloader" async></script>
+    <link rel="stylesheet" href="https://ig.ft.com/visual-vocabulary/styles.css"/>
+    <link rel="stylesheet" href="./grouped-symbol-column.css"/>
+    <link rel="stylesheet" href="//www.ft.com/__origami/service/build/v2/bundles/css?modules=o-fonts@^2.2.0" />
+    <style>
+    .saveable button{ display: block; }
+    </style>
```

Comment:
This is probably superfluous as it should be added to global styles (if it's not there already).